src/common/ceph_context.cc

```diff
@@ -356,6 +360,7 @@ class LogObs : public md_config_obs_t {
 };
 
 
+namespace TOPNSPC::common {
```

Comment:
why?

Reply:
chage it to ceph::common,  need add the namespace so friend class can access private member. such as _cct->_heartbeat_map works.